packages/smooth_app/lib/helpers/product_cards_helper.dart

```diff
@@ -22,25 +22,42 @@ Widget buildProductTitle(
     );
 
 String getProductNameAndBrands(
-    Product product, AppLocalizations appLocalizations) {
-  final String name =
-      product.productName?.trim() ?? appLocalizations.unknownProductName;
-  final String brands = product.brands?.trim() ?? appLocalizations.unknownBrand;
+  final Product product,
+  final AppLocalizations appLocalizations,
+) {
+  final String name = getProductName(product, appLocalizations);
+  final String brands = getProductBrands(product, appLocalizations);
   return '$name, $brands';
 }
 
-String getProductName(Product product, AppLocalizations appLocalizations) =>
-    product.productName ??
-    product.productNameInLanguages?[ProductQuery.getLanguage()] ??
+/// Returns a trimmed version of the string, or null if null or empty.
+String? _clearString(final String? string) {
```

Comment:
This kind of feature may be useful elsewhere in the app.
I suggests to move-it to a helper-like class, or even better as an extension